### ISSUE TITLE
Fix rspec Docker directories, Improve pull up and tear down process

### ISSUE THREAD

**Issue by franz-josef-kaiser** [franz-josef-kaiser/Docker-Alpine-Nodejs#7]
Type of issue: Bug report
### Expected/Desired behavior
- Rspec should fetch the `Dockerfile` as base.
- The pull up process should create a container
- The pull up process should log details about the environment for easier developing and better bug reports
- The tear down process should destroy the container
- The specs should test the OS version and the installed packages
### Actual/Current behavior

It [does not](https://travis-ci.org/franz-josef-kaiser/Docker-Alpine-Nodejs/builds/115354184#L245) as the build path is wrong:

```
Failure/Error: image = Docker::Image.build_from_dir( '.' )
```

For the rest, we need actual specs and find out how to check for the Alpine Linux version. It seems that [there is a patch for that](http://git.alpinelinux.org/cgit/aports/plain/main/xe-guest-utilities/identify_alpine.patch) in the Alpine upstream.
### Proposed fix

Move to parent directory to find `Dockerfile`.

**Comment by franz-josef-kaiser** [comment]
Currently Travis CI errors out with a `permission denied` for `tests/install_dependencies.sh` script. As seen on SO, the simple fix is to change permissions locally and push to GH again.

```
chmod ugo+x tests/install_dependencies.sh
```

**Comment by franz-josef-kaiser** [comment]
The `tests/install_dependencies.sh` script uses `docker -d`, which gets deprecated with one of the future versions. Needs to get changed to `docker deamon` instead. Fixed with 2937599

**Comment by franz-josef-kaiser** [comment]
As parent directory look up for `Dockerfile` did not work out, I now changed the `tests` dir to `ci` and moved the spec file to root. Fixed with ebc5602
